src/cmap/errors.ts

```diff
@@ -12,9 +12,12 @@ export class PoolClosedError extends MongoDriverError {
 
   constructor(pool: ConnectionPool) {
     super('Attempted to check out a connection from closed connection pool');
-    this.name = 'MongoPoolClosedError';
     this.address = pool.address;
   }
+
+  get name(): string {
+    return 'PoolClosedError';
```

Comment:
are the two renamings in this file intentional? these could potentially be breaking changes

Reply:
This was unintentional, but this did highlight another issue. The test should assert that the `name` prop matches the actual name of the class, but this isn't true for `PoolClosedError` and `WaitQueueTimeoutError` as both of their name props add the "Mongo" prefix. For now, I've imported both of those errors and aliased them with their "Mongo" prefix, but it's only a temporary fix.